src/libs/ReportUtils.js

```diff
@@ -206,7 +200,7 @@ function sortReportsByLastRead(reports) {
  */
 function canEditReportAction(reportAction) {
     return (
-        reportAction.actorEmail === sessionEmail &&
+        reportAction.actorAccountID === sessionAccountID &&
```

Comment:
FYI, in [this PR](https://github.com/Expensify/App/pull/21822) I got rid of sessionAccountID and replaced it with currentUserAccountID. As you suggested 😉

Reply:
Aah nice thanks for the reminder :D